### PR TITLE
Update springer-mathphys styles

### PR DESCRIPTION
**Pull request by damnation333**
via https://forums.zotero.org/profile/3932165/annadci
- added to include DOI for journals.
- removed the dot at the end

### CODE REVIEW

**Comment by damnation333** [comment]
@adam3smith 
I'll never get this GitHub thingamajig....
I've edited in my damnation333 repo and then clicked on commit to master branch. i assume that was wrong? https://github.com/damnation333/styles/blob/master/springer-mathphys-brackets.csl

**Comment by adam3smith** [comment]
yeah, you need to make the change in your patch-22 branch

**Comment by damnation333** [comment]
I think that was unsuccesfull: https://github.com/damnation333/styles/pull/1

**Comment by adam3smith** [comment]
no sorry, that went the wrong way. If you'd like to figure it out, I can walk you through that. Otherwise I'm also just happy to make the change for you.

**Comment by damnation333** [comment]
NO, I need to learn this. Where the heck do I find branch-22 of my repository? I see it mentioned up there, but you can't click on it.

**Comment by adam3smith** [comment]
Two ways:
1) Go to https://github.com/damnation333/styles and at the top left, where it says "Branch: Master" click on the arrow next to it and select patch-22

2) Go to https://github.com/damnation333/styles/tree/patch-22 directly. Change the last part of the URL for different branches, obviously.

**Comment by damnation333** [comment]
ahhhhhhhhhhh!!
Thanks. That was the missing link. would've never looked there for the patches.

File: springer-mathphys-author-date.csl [hunk 1]
```diff
@@ -114,7 +115,7 @@
       <key macro="author-in-citation"/>
       <key macro="issued-sort" sort="ascending"/>
     </sort>
-    <layout suffix=".">
+    <layout>
```
Comment (adam3smith) [inline springer-mathphys-author-date.csl]: None of the references have a period at the end or just not the ones with DOI? For the latter, we'd need to find a better solution. I also wonder if it wouldn't make sense to put the access macro in there independent of item type now that we increasingly see books & chapters with DOI.

Comment (damnation333) [inline springer-mathphys-author-date.csl]: None of them. Must have been a mistake in the old style (or they changed it).

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**